agents/backport_agent.py

```diff
@@ -86,7 +86,8 @@ def get_instructions() -> str:
          Use `rpmlint <PACKAGE>.spec` to validate your changes and fix any new issues.
 
       6. Run `centpkg --name=<PACKAGE> --namespace=rpms --release=<DIST_GIT_BRANCH> prep` to see if the new patch
-         applies cleanly.
+         applies cleanly. When `prep` command finishes with "exit 0", it's a success. Ignore errors from
+         libtoolize that warn about newer files: "use '--force' to overwrite".
```

Comment:
autotools/libtool are always PITA, I'm sure we'll encounter more similar issues.